addons/mail/static/src/views/web/fields/kanban_activity/kanban_activity.js

```diff
@@ -15,7 +15,11 @@ export class KanbanActivity extends Component {
             selection: [("warning", "Alert"), ("danger", "Error")],
         },
         { name: "activity_exception_icon", type: "char" },
-        { name: "activity_state", type: "selection" },
+        {
+            name: "activity_state",
+            type: "selection",
+            selection: [("overdue", "Overdue"), ("today", "Today"), ("planned", "Planned")],
+        },
```

Comment:
Can you explain why do we need to do that? 🧐

Reply:
We had a traceback when adding the default kanban view to the sub-tasks/blocked by x2many record, the problem came from the widget kanban_activity used to display the activity_ids on the task kanban card.
It had fieldDependencies that were not present in the view.

https://discord.com/channels/678381219515465750/687338039717920792/1185135051806081065

This was reported to the Framework JS team and fixed in the following PRs:
https://github.com/odoo/odoo/pull/146435
https://github.com/odoo/odoo/pull/148351

This is no longer needed to prevent a traceback